### PR TITLE
Weak ext ext and paths in contractible types

### PR DESCRIPTION
**Pull request by jonalfcam**
I added a proof that in contractible types paths can be identified. I also had two easy lemmas in the extension extensionality section

### CODE REVIEW

**Comment by emilyriehl** [comment]
Hi @jonalfcam. Your first term already exists and is called [eq-is-contr](https://rzk-lang.github.io/sHoTT/hott/06-contractible.rzk/#define:eq-is-contr).

File: src/hott/06-contractible.rzk.md [hunk 1]
```diff
@@ -486,3 +486,70 @@ A type is contractible if and only if it has singleton induction.
...
+    ( left-inverse-concat A (center-contraction A is-contr-A) x (homotopy-contraction A is-contr-A x)) 
+    ( y) 
+    ( p) 
+
+```
+
+Finally, in a contractible type any two paths between the same end points are equal. There are many possible proofs of this (e.g. identifying contractible types with the unit type where it is more transparent), but we proceed by gluing together the two identifications to the out and back path.
+
+```rzk
+#define paths-eq-is-contr
```
Comment (emilyriehl) [inline src/hott/06-contractible.rzk.md:535]: I might call this `eq-paths-is-contr`.

Comment (jonalfcam) [reply src/hott/06-contractible.rzk.md:535]: Great! I'll make these changes.

**Comment by emilyriehl** [comment]
I like leaving the new terms in the contractible types file in the new section. Just reference the existing definition of `eq-is-contr` from the contractible types data section. Otherwise this looks ready to merge.

File: src/hott/06-contractible.rzk.md (hunk 1 above)
Comment (jonalfcam) [reply src/hott/06-contractible.rzk.md:535]: I jumped the gun! Will change to this name and push again.

Comment (jonalfcam) [reply src/hott/06-contractible.rzk.md]: Done. This has the new suggestions for names as well as the references to `eq-is-contr`

**Review by jonalfcam — COMMENTED** [review]
changes to names

**Review by emilyriehl — APPROVED** [review]

**Pull request merged**